### PR TITLE
Fixing deps + adding checks

### PR DESCRIPTION
**Pull request by AndreaCensi**
## Description

Fixing the wrong library being imported (reprep → reprep-z6).
Also, not sure why this image is the right place to import it.

Adding `pip check` consistency check.

Might break downstream.

## Type of change

Please mark the relevant options with X:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change is constrained only to one package, any interfaces to nodes from other packages are maintained
- [ ] This change is constrained only to packages in this repository, any interfaces to nodes from other repositories are maintained
- [ ] This change required a documentation update
- [ ] This change is only a documentation update
- [ ] This change restructures the repository (at level higher than package level)
- [ ] This change requires a change in the Duckiebook (if yes, please describe and, if possible, make a pull request for an edit in the book and link it here)

## Tests

Please describe all tests ran and their outcome. It is highly recommended that you at least do the tests listed by default, even if you only changed the documentation.

- [ ] Made the Duckiebot move with keyboard control
- [ ] Visually checked the published camera stream
- [ ] Verified that the lane following demo works
- [ ] Verified that the indefinite navigation demo works

## Style compliance
Please make sure that all your changes conform to the Duckietown (code style)[link] and code (documentation style)[link] guidelines:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I checked that this pull request is compliant with the Duckietown code style
- [x] I checked that this pull request is compliant with the Duckietown code documentation style

## Next steps

@afdaniele some of downstream images might break

## FOR THE REVIEWERS
__IMPORTANT:__ DO NOT CHANGE THIS SECTION!

- [ ] Functionality (is the change necessary and is it implemented in the most appropriate way): @liampaull or @afdaniele
- [ ] Integration testing (verify the tests and run additional if necessary): @?
- [ ] Code style compliance: @gibernas
- [ ] Documentation compliance: @aleksandarpetrov

_To the reviewers:_ If you deem necessary, request code edits, more tests, documentation update, or additional reviewing. __Do not review your own code!__

_PR Template, last edit 23 Aug 2019_

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -51,6 +51,9 @@ COPY ./dependencies-apt.txt "${REPO_PATH}/"
 RUN dt-apt-install ${REPO_PATH}/dependencies-apt.txt
 
 # install python3 dependencies
+# run `pip check` to make sure situation consistent
+# before we install our dependencies
+RUN pip check
```
Comment (afdaniele) [inline Dockerfile:56]: @AndreaCensi shouldn't this be `pip3`?

File: Dockerfile [hunk 2]
```diff
@@ -84,3 +87,7 @@ LABEL org.duckietown.label.module.type="${REPO_NAME}" \
 # <==================================================
 
 ENV DUCKIETOWN_ROOT="${SOURCE_DIR}"
+
+RUN pipdeptree
+RUN pip list
+RUN pip check
```
Comment (afdaniele) [inline Dockerfile:93]: @AndreaCensi same here, `pip3`?

File: Dockerfile (hunk 1 above)
Comment (AndreaCensi) [reply Dockerfile:56]: Sure - (we still have python2 stuff?)

Comment (afdaniele) [reply Dockerfile:56]: Nope

**Pull request merged**